python/pyabacus/CMakeLists.txt

```
@@ -32,6 +32,10 @@ add_subdirectory(${ABACUS_SOURCE_DIR}/module_base ${BASE_BINARY_DIR})
 set(ORB_BINARY_DIR "${PROJECT_SOURCE_DIR}/build/orb")
 set(ENABLE_LCAO ON)
 add_subdirectory(${ABACUS_SOURCE_DIR}/module_basis/module_ao ${ORB_BINARY_DIR})
+# set RPATH
+set(CONDA_PREFIX $ENV{CONDA_PREFIX})
+set(CMAKE_INSTALL_RPATH "${CONDA_PREFIX}/lib/python3.8/site-packages/pyabacus")
```

@jieli-matrix does `python3.8` symlinked to `python` folder?
```suggestion
set(CMAKE_INSTALL_RPATH "${CONDA_PREFIX}/lib/python/site-packages/pyabacus")
```

---

Fix in the latest commit!

---

> Fix in the latest commit!

@jieli-matrix Thanks! And don't forget to resolve the conversation after fixing.